e2e/cypress/integration/search/post_search_display_spec.js

```diff
@@ -52,7 +52,7 @@ describe('Search', () => {
 
         // # Click on searchbox
         cy.get('#searchbarContainer').should('be.visible').within(() => {
-            cy.get('#searchBox').click();
+            cy.get('#searchBox').click({force: true});
```

Comment:
Looks like this one still failed - https://automation-dashboard.vercel.app/cycles/a9e52661-f080-4d3a-9ba6-3dc197a0efa4